### DISCUSSION TITLE
Removing copyright and adding Creative Commons licence

### DISCUSSION THREAD

**Discussion by DamoM73** [pradyunsg/furo#612] [Questions]
Hello

I come seeking help. I'm secondary school digital technologies teacher, and I have just used Sphinx to create a Python Introduction course for my students.

I would like to publish the site under a Creative Commons licence, but I cannot work out how to replace the copyright. I've looked through the Sphinx documentation, but it is just out of my reach. I have tried some tutorials but I haven't managed success.

I was wondering if anyone could point me in the right direction.

**Comment by jdknight** [comment]
I do not think there is an ideal answer here. Sphinx themes (Furo included) typically embed the copyright prefix to work with Sphinx's [`copyright`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright) configuration value. Maybe an enhancement for Sphinx would be to support an option like `project_license` which allows projects to completely override the full license text (but that is outside of this theme's scope).

You could try to extend the Furo theme and provide an override for the `footer` block, but that might be too much effort than what you are looking for.

An alternative and extremely hacking way to dealing with this is to use CSS to override the copyright text. This involves injecting a custom style sheet in the project's `conf.py`:

```
def setup(app):
    app.add_css_file('theme_overrides.css')
```

And having `_static/theme_overrides.css` doing something such as:

```
.copyright {
    visibility: hidden;
}

.copyright:before {
    content: 'Text is available under the Creative Commons Attribution-ShareAlike License 3.0.';
    visibility: visible;
}
```

**Reply by DamoM73** [reply]
Thanks for the suggestion.

It didn't quite work. 

I've decided to add a licence page instead which explains the situation.

Thanks for your help.

**Comment by pradyunsg** [comment]
This is not legal advice and I'm absolutely not a lawyer, but...

You still have a copyright involved, even if you end up using a very "relaxed" license on the content. Whether that stays with you or gets assigned to someone else is a different discussion.

I believe you can omit the `copyright = ...` in conf.py, and Furo will gracefully omit the copyright line from the footer. If not, I likely broke that at some point and would be happy to fix it.

I'd say the easiest way to handle things here would be to add a license page, which shows up in the sidebar (similar to https://installer.pypa.io/en/stable/license/) and remove the copyright from conf.py.
